src/scss/components/_video-card.scss

```diff
@@ -21,6 +21,34 @@
         border-radius: calc-em(10, 16);
     }
 
+    &__icon-left {
+        &:before {
+            left: calc-em(20, 36);
+            right: auto;
+        }
+    }
+
+    &__icon-white {
+        &:before {
```

Comment:
use same hierarchy used for the normal play, removing the !important.
&__icon-white {
picture {
        &:before {

Reply:
not sure about this one, I just removed the icon white class , so its using normal play icon